app/adapters/collection.ts

```diff
@@ -0,0 +1,21 @@
+import OsfAdapter from './osf-adapter';
+
+export default class Collection extends OsfAdapter.extend({
+    buildURL(_: any, __: any, ___: any, requestType: string): string {
+        // Embed linked_nodes
+        const base: string = this._super(...arguments);
+        if (['createRecord', 'updateRecord', 'deleteRecord'].indexOf(requestType) === -1) {
```

Comment:
The array should be declared outside of the function scope, but defining an enum would be better